src/workers/postUpdated.ts

```diff
@@ -107,64 +108,73 @@ const createPost = async ({
   mergedKeywords,
   questions,
 }: CreatePostProps) => {
-  const existingPost = await entityManager
-    .getRepository(Post)
-    .createQueryBuilder()
-    .select('id')
-    .where(
-      'url = :url or url = :canonicalUrl or "canonicalUrl" = :url or "canonicalUrl" = :canonicalUrl',
-      { url: data.url, canonicalUrl: data.canonicalUrl },
-    )
-    .getRawOne();
-  if (existingPost) {
-    logger.info({ data }, 'failed creating post because it exists already');
-    return null;
-  }
+  const meter = opentelemetry.metrics.getMeter('api-bg');
```

Comment:
Coming from the main handler function or from outside in general?

Reply:
main handler function I think. On the API side it usually comes from the `req` object but it doesn't hold here in the bg worker

Reply:
> It’s generally recommended to call `getMeter` in your app when you need it rather than exporting the meter instance to the rest of your app. This helps avoid trickier application load issues when other required dependencies are involved.
> [source](https://opentelemetry.io/docs/instrumentation/js/manual/#acquiring-a-meter)